mxcube3/ui/components/SampleView/SampleImage.jsx

```diff
@@ -308,7 +308,7 @@ export default class SampleImage extends React.Component {
     const { sendGoToBeam } = sampleActions;
 
     // Only move to beam if the click was done directly on the canvas.
-    if (e.target.tagName === 'CANVAS') {
+    if (e.target.tagName === 'CANVAS' && e.shiftKey) {
```

Comment:
why adding the shift for this? I would rather keep without...

Reply:
Because it happens that users accidentally double click on the canvas and then the sample is moved. It also happens that one during 3-click centering clicks a bit fast and then accidentally also send move to beam. Thats why we though it would be better with a modifier for move to beam